openlcs/libs/sc_handler.py

```diff
@@ -113,3 +123,69 @@ def get_container_components(self, srpm_dir, misc_dir, sc_nvr):
                     'CONTAINER_IMAGE': container_component
         }
         return components
+
+    def collate_source_container_srpm(self):
+        """
+        Unpack the source RPM in source container.
+        """
+        source_dir = os.path.join(self.dest_dir, "blobs", "sha256")
+        srpm_link_dir = os.path.join(self.dest_dir, 'rpm_dir')
+        if os.path.exists(srpm_link_dir):
+            srpm_names = os.listdir(srpm_link_dir)
+            for srpm_name in srpm_names:
+                try:
+                    soft_link_path = os.path.join(srpm_link_dir, srpm_name)
+                    sha256_str = os.path.basename(os.readlink(soft_link_path))
+                    sha256_path = os.path.join(source_dir, sha256_str)
+                    dest_path = os.path.join(srpm_link_dir, srpm_name)
```

Comment:
The dest_path is the same as the soft_link_path? If so what's the meaning of the dest_path?

Reply:
Yes, this is a mistake, removed one of them, only use soft_link_path